src/java/src/main/java/ai/onnxruntime/genai/Model.java

```diff
@@ -6,6 +6,14 @@
 public final class Model implements AutoCloseable {
   private long nativeHandle;
 
+  public Model(String modelPath, String qnnLibSearchPath) throws GenAIException {
```

Comment:
I don't feel like adding public interfaces like this. But I don't have better ideas. @skottmckay  any thought?